src/main/java/core/basesyntax/model/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.model.controller;
+
+import core.basesyntax.model.dao.BetDao;
+import core.basesyntax.model.dao.UserDao;
+import core.basesyntax.model.lib.Inject;
+import core.basesyntax.model.model.Bet;
+import core.basesyntax.model.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+
+    @Inject
+    private BetDao betDao;
+
+    @Inject
+    private UserDao userDao;
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            System.out.println("Enter value and risk for your bet.");
+            String command = scanner.nextLine();
+
+            if (command.equalsIgnoreCase("q")) {
+                System.out.println("all bets " + betDao.getAll()
+                        + "\n" + "all users" + userDao.getAll());
+                return;
+            }
+
```

Comment:
there are too many empty lines in your code

Reply:
done